### ISSUE TITLE
torch的版本有限制吗？训练时报错！求助解答

### ISSUE THREAD

**Issue by findlyhl** [hpcaitech/Open-Sora#33]
之前那个diffusers的问题没有了，出现了新的问题。求助解答
torch的版本有限制吗？我用的版本是torch==2.1.2+cu121
现在的报错信息：
TypeError: cannot pickle 'torch._C._distributed_c10d.ProcessGroup' object
[2024-03-06 10:13:06,886] torch.distributed.elastic.multiprocessing.api: [ERROR] failed (exitcode: 1) local_rank: 0 (pid: 52488) of binary: /root/miniconda3/bin/python

我怀疑是torch版本的问题

**Comment by cxiaolong** [comment]
I solved this problem reference by issue#28

`ema = DiT_models[args.model](**model_kwargs).to(get_current_device())`
in replacement of the line with `ema = deepcopy(model).to(device)`

**Comment by ver217** [comment]
> I solved this problem reference by issue#28
> 
> `ema = DiT_models[args.model](**model_kwargs).to(get_current_device())` in replacement of the line with `ema = deepcopy(model).to(device)`

These two methods are not equivalent. If not using deepcopy, we should sync ems's params from origin model after initializing ema.

**Comment by cxiaolong** [comment]
```
# Step 4: Create DiT and EMA
model = DiT_models[args.model](**model_kwargs))
ema = deepcopy(model).to(device)
patch_size = model.patch_size
model.to(get_current_device())
requires_grad(ema, False)
```

**Comment by cheliu-computation** [comment]
> > I solved this problem reference by issue#28
> > `ema = DiT_models[args.model](**model_kwargs).to(get_current_device())` in replacement of the line with `ema = deepcopy(model).to(device)`
> 
> These two methods are not equivalent. If not using deepcopy, we should sync ems's params from origin model after initializing ema.

Thanks for your explaination, would you advice me to solve this issue without change this line?
